src/aead.rs

```diff
@@ -269,6 +269,8 @@ impl<N: NonceSequence> core::fmt::Debug for SealingKey<N> {
 
 impl<N: NonceSequence> SealingKey<N> {
     /// Deprecated. Renamed to [`seal_in_place_append_tag()`].
+    ///
+    /// [`seal_in_place_append_tag()`]: #method.seal_in_place_append_tag
```

Comment:
There must be a more compact, less repetitive, way to do this? This isn't a sustainable way to do it.

Reply:
Yup. Fixed.